### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by ue71603**
We finally should do a README for main.

### CODE REVIEW

**Comment by ue71603** [comment]
I will do it for develop also...

File: README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+
+# OJP - *O*pen API for distributed *J*ourney *P*lanning
+
+## Content
+This repository contains the XSD, examples and documentation tables for the 
+"**O**pen API for distributed **J**ourney **P**lanning" published as 
+*"CEN/TS 17118:2017: "Intelligent transport systems - Public transport - Open API for distributed journey planning"*.
```
Comment (skinkie) [inline README.md:7]:
```suggestion
This repository contains the XML Schema, examples and documentation tables for the "**O**pen API for distributed **J**ourney **P**lanning" published as *"CEN/TS 17118:2017: "Intelligent transport systems - Public transport - Open API for distributed journey planning"*.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+* Information about fares for trips
+* Changing of trips
+* Refining of trips
+* Availability for trip legs
+* Information about lines
+
+OJP systems interacting with each other additionally use:
+* Getting information about Exchange Points (places when a Trip switches from one OJP system to another)
+* Using a multi point trip service (n origins and m destinations)
+
```
Comment (skinkie) [inline README.md:25]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+## Standard document
+OJP is a CEN standard. Obtain a copy of the standard document
+["CEN/TS 17118:2017: "Intelligent transport systems - Public transport - Open API for distributed journey planning"](https://standards.cen.eu/dyn/www/f?p=204:110:0::::FSP_PROJECT:62236&cs=1985DBD613F25D179FB65A73B0FDA4DB7)
+at [CEN](https://www.cen.eu).
```
Comment (skinkie) [inline README.md:30]:
```suggestion
OJP is a CEN standard. Obtain a copy of the standard document ["CEN/TS 17118:2017: "Intelligent transport systems - Public transport - Open API for distributed journey planning"](https://standards.cen.eu/dyn/www/f?p=204:110:0::::FSP_PROJECT:62236&cs=1985DBD613F25D179FB65A73B0FDA4DB7) at [CEN](https://www.cen.eu).
```

**Review by skinkie — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+* Searching for trips
+* Information about a given trip
+* Information about fares for trips
+* Changing of trips
+* Refining of trips
+* Availability for trip legs
+* Information about lines
+
+OJP systems interacting with each other additionally use:
+* Getting information about Exchange Points (places when a Trip switches from one OJP system to another)
```
Comment (sgrossberndt) [inline README.md:21]: Trip => trip

File: README.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+You can contribute by:
+* Pointing out [issues](https://github.com/VDVde/OJP/issues)
+* Creating pull requests: Use the [development documentation](https://github.com/VDVde/OJP/wiki/WG:-:-Working-with-this-repository)
+
+### Branches
+* The master branch is currently set to OJP 1.0.1
+* Development occurs currently in the development branch: https://github.com/VDVde/OJP/tree/develop.
+* We work with release-branches
+* PR should always be created with feature/<id> as a branch.
```
Comment (sgrossberndt) [inline README.md:47]: You did not create a feature branch for this change itself..

File: README.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+* Information about lines
+
+OJP systems interacting with each other additionally use:
+* Getting information about Exchange Points (places when a Trip switches from one OJP system to another)
+* Using a multi point trip service (n origins and m destinations)
+
+## Standard document
+OJP is a CEN standard. Obtain a copy of the standard document ["CEN/TS 17118:2017: "Intelligent transport systems - Public transport - Open API for distributed journey planning"](https://standards.cen.eu/dyn/www/f?p=204:110:0::::FSP_PROJECT:62236&cs=1985DBD613F25D179FB65A73B0FDA4DB7) at [CEN](https://www.cen.eu).
+
+OJP is part of the [Transmodel](https://www.transmodel-cen.eu/)) family of standards together with [NeTEx](https://netex-cen.eu/), [SIRI](https://www.transmodel-cen.eu/siri-standard/)  and OPRA. It also contains elements from [DATEX II](https://datex2.eu/).
```
Comment (sgrossberndt) [inline README.md:27]: two spaces

File: README.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+* Creating pull requests: Use the [development documentation](https://github.com/VDVde/OJP/wiki/WG:-:-Working-with-this-repository)
+
+### Branches
+* The master branch is currently set to OJP 1.0.1
+* Development occurs currently in the development branch: https://github.com/VDVde/OJP/tree/develop.
+* We work with release-branches
+* PR should always be created with feature/<id> as a branch.
+
+### Wiki
+OJP uses the [wiki](https://github.com/VDVde/OJP/wiki) on github for some resources and examples.
```
Comment (sgrossberndt) [inline README.md:50]: github => GitHub

File: README.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+* Getting information about Exchange Points (places when a Trip switches from one OJP system to another)
+* Using a multi point trip service (n origins and m destinations)
+
+## Standard document
+OJP is a CEN standard. Obtain a copy of the standard document ["CEN/TS 17118:2017: "Intelligent transport systems - Public transport - Open API for distributed journey planning"](https://standards.cen.eu/dyn/www/f?p=204:110:0::::FSP_PROJECT:62236&cs=1985DBD613F25D179FB65A73B0FDA4DB7) at [CEN](https://www.cen.eu).
+
+OJP is part of the [Transmodel](https://www.transmodel-cen.eu/)) family of standards together with [NeTEx](https://netex-cen.eu/), [SIRI](https://www.transmodel-cen.eu/siri-standard/)  and OPRA. It also contains elements from [DATEX II](https://datex2.eu/).
+
+## Releases
+* [OJP 1.0 branch](https://github.com/VDVde/OJP/tree/release/1.0)
```
Comment (sgrossberndt) [inline README.md:30]: I'd omit the branches from the releases section, we have the branches section for this:

```
## Releases
* [OJP v1.0](https://github.com/VDVde/OJP/releases/tag/v1.0): The version described in CEN/TS 17118:2017
* [OJP v1.0.1](https://github.com/VDVde/OJP/releases/tag/v1.0.1): Bug fix release
* OJP v2.0 is in preparation in the [develop branch](https://github.com/VDVde/OJP/tree/develop).
```

File: README.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+
+Download the XSD schema files and examples in the git repository found at https://github.com/VDVde/OJP and take a look at the [documentation tables](https://vdvde.github.io/OJP/index.html).
+
+You can contribute by:
+* Pointing out [issues](https://github.com/VDVde/OJP/issues)
+* Creating pull requests: Use the [development documentation](https://github.com/VDVde/OJP/wiki/WG:-:-Working-with-this-repository)
+
+### Branches
+* The master branch is currently set to OJP 1.0.1
+* Development occurs currently in the development branch: https://github.com/VDVde/OJP/tree/develop.
```
Comment (sgrossberndt) [inline README.md:45]: in the [develop branch](https://github.com/VDVde/OJP/tree/develop)

File: README.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+## Github Repository
+
+Download the XSD schema files and examples in the git repository found at https://github.com/VDVde/OJP and take a look at the [documentation tables](https://vdvde.github.io/OJP/index.html).
+
+You can contribute by:
+* Pointing out [issues](https://github.com/VDVde/OJP/issues)
+* Creating pull requests: Use the [development documentation](https://github.com/VDVde/OJP/wiki/WG:-:-Working-with-this-repository)
+
+### Branches
+* The master branch is currently set to OJP 1.0.1
```
Comment (sgrossberndt) [inline README.md:44]: master => main

File: README.md [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+
+OJP is part of the [Transmodel](https://www.transmodel-cen.eu/)) family of standards together with [NeTEx](https://netex-cen.eu/), [SIRI](https://www.transmodel-cen.eu/siri-standard/)  and OPRA. It also contains elements from [DATEX II](https://datex2.eu/).
+
+## Releases
+* [OJP 1.0 branch](https://github.com/VDVde/OJP/tree/release/1.0)
+  * [OJP 1.0](https://github.com/VDVde/OJP/releases/tag/v1.0): The version described in CEN/TS 17118:2017
+  * [OJP 1.0.1](https://github.com/VDVde/OJP/releases/tag/v1.0.1): Bug fix release
+* OJP 2.0 in preparation. See the following [branch](https://github.com/VDVde/OJP/tree/develop).
+
+## Github Repository
```
Comment (sgrossberndt) [inline README.md:35]: Github repository => GitHub repository

**Review by sgrossberndt — CHANGES_REQUESTED** [review]

File: README.md (hunk 11 above)
Comment (ue71603) [inline README.md:35]:
```suggestion
## GitHub Repository
```

Comment (ue71603) [reply README.md:35]: done

File: README.md (hunk 10 above)
Comment (ue71603) [inline README.md:44]:
```suggestion
* The main branch is currently set to OJP 1.0.1
```

Comment (ue71603) [reply README.md:44]: done

File: README.md (hunk 9 above)
Comment (ue71603) [inline README.md:45]:
```suggestion
* Development occurs currently in [the develop branch](https://github.com/VDVde/OJP/tree/develop).
```

File: README.md (hunk 5 above)
Comment (ue71603) [reply README.md:47]: I know. Last one :-)

File: README.md (hunk 7 above)
Comment (ue71603) [inline README.md:50]:
```suggestion
OJP uses the [wiki](https://github.com/VDVde/OJP/wiki) on GitHub for some resources and examples.
```

Comment (ue71603) [reply README.md:50]: done

File: README.md (hunk 5 above)
Comment (ue71603) [inline README.md:47]:
```suggestion
* PR should always be created in a branch with the name "feature/<title>".
```

File: README.md [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+
+You can contribute by:
+* Pointing out [issues](https://github.com/VDVde/OJP/issues)
+* Creating pull requests: Use the [development documentation](https://github.com/VDVde/OJP/wiki/WG:-:-Working-with-this-repository)
+
+### Branches
+* The main branch is currently set to OJP 1.0.1
+* Development occurs currently in [the develop branch](https://github.com/VDVde/OJP/tree/develop).
+* We work with release-branches
+* PR should always be created in a branch with the name "feature/<title>".
```
[the remark this answers is not in the document]
Comment (trurlurl) [reply README.md:45]: Would this correspond to what is meant by *with the name "feature/<title>"*?

**Comment by ue71603** [comment]
redo it based on the latest version

**Pull request closed without merging**